otel-extension/src/main/scala/akka/ActorGraphInterpreterProcessEventAdvice.scala

```diff
@@ -0,0 +1,35 @@
+package akka
+
+import _root_.io.scalac.mesmer.agentcopy.akka.stream.impl.ActorGraphInterpreterOtelDecorator
```

Comment:
I think you can replace "agentcopy" to "otelextension.instrumentations"

Reply:
Yeah, I chose the name just to have something different than `agent ` to be sure we load the correct classes 🙂 we can move it to `otelextension.instrumentations`